### PR TITLE
Some small improvements

### PR DESCRIPTION
**Pull request by mina86**

### CODE REVIEW

File: nfsn-cron.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
-set -o errexit
+set -eu
```
Comment (Celti) [inline nfsn-cron.sh:2]: I prefer to use long names for shell options, for clarity. This is a good idea, though.

File: nfsn-cron.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
-cd $(dirname $0)
+cd "$(dirname "$0")"
```
Comment (Celti) [inline nfsn-cron.sh:5]: Given that this is NFSN, you would have to be braindead for this to be anywhere besides `/home/private/`, and if you deliberately change the names of your shell scripts to include spaces... well, that's gone right past braindead. Still, you can never be _too_ careful, right? :-p

File: nfsn-hook.sh [hunk 3]
```diff
@@ -25,4 +26,4 @@ function unchanged_cert {
     echo " + Certificate for ${DOMAIN} unchanged."
 }
 
-HANDLER=$1; shift; $HANDLER $@
+"$@"
```
Comment (Celti) [inline nfsn-hook.sh:29]: Again, for clarity I prefer the longer version, though `$@` should be quoted for sanity.

File: nfsn-setup.sh [hunk 4]
```diff
@@ -19,7 +19,25 @@ nfsn -s list-aliases > letsencrypt.sh/domains.txt
...
+       Hour:                 %d
+       Day of Week:          Every
+       Date:                 *
+
+   The certificates will be renewed only when needed so it’s safe to schedule
+   the task to run daily.
+' \
+       "${user_site%_*}" "$NFSN_SITE_NAME" \
+       "$BASH" "$(realpath nfsn-cron.sh)" \
+       "$(( $RANDOM % 24 ))"
```
Comment (Celti) [inline nfsn-setup.sh:43]: Now this, this I like, though I'll need to add a note about cron being broken until `/usr/local/bin/nfsn` is updated.

Comment (mina86) [inline nfsn-setup.sh:43]: On Fri, Jun 10 2016, Patrick Burroughs wrote:

> Now this, this I like, though I'll need to add a note about cron being
> broken until `/usr/local/bin/nfsn` is updated.

Oh? Is that why I’m getting:

```
nfsn FATAL: Unable to connect to 192.168.1.1:1
```

## 

Best regards
ミハウ “𝓶𝓲𝓷𝓪86” ナザレヴイツ
«If at first you don’t succeed, give up skydiving»

**Comment by Celti** [comment]
Indeed, that's why you're getting that error. jdw is trying to figure out how to balance authentication and convenience; until then, I've added a workaround to nfsn-cron.sh and a warning to nfsn-setup.sh — as well as merging your PR (with cleanups).

Thanks for the contribution!

**Pull request merged**